### ISSUE TITLE
[BUG]: Windows Defender Detects Uninstaller as a Virus

### ISSUE THREAD

**Issue by hl2guide** [undergroundwires/privacy.sexy#114]
### Description

Windows Defender detects the uninstaller as a virus (Trojan:Win32/Bulta!rfn).

### Reproduction steps

1. Install the app on Windows with Windows Defender active
2. Notice Uninstaller has been removed (message)

### Expected behavior

No detection.

**Comment by undergroundwires** [comment]
Thank you for the report! I can reproduce this.

TLDR; I'll issue a patch, but not too soon, it's a false positive caused by upstream packager, we can just ignore it.

This is caused by our external dependency `electron-builder` that's used to create the NSIS installer. I see that it only happens in latest patch released yesterday where `electron-builder` version was bumped from `22.9.1` ([package-lock.json in 0.11.2](https://raw.githubusercontent.com/undergroundwires/privacy.sexy/0.11.2/package-lock.json)) to latest `22.14.5` ([package-lock.json in 0.11.3](https://raw.githubusercontent.com/undergroundwires/privacy.sexy/0.11.3/package-lock.json)) . This dependency bump was unintended as side effect of introducing new (ESlint) and bumping other development dependencies (Vue CLI).

According to electron-userland/electron-builder#4793 updating this dependency to pre-release version fixes the issue. Pr we could downgrade it to working version to not risk using beta software. I will issue a new patch fixing this along with some other minor improvements. In meantime I think we should flag those warnings as false positives so Microsoft Defender draw better conclusions.

**Comment by undergroundwires** [comment]
Thank you for the report! I can reproduce this.

TLDR; I'll issue a patch, but not too soon, it's a false positive caused by upstream packager, we can just ignore it.

This is caused by our external dependency `electron-builder` that's used to create the NSIS installer. I see that it only happens in latest patch released yesterday where `electron-builder` version was bumped from `22.9.1` ([package-lock.json in 0.11.2](https://raw.githubusercontent.com/undergroundwires/privacy.sexy/0.11.2/package-lock.json)) to latest `22.14.5` ([package-lock.json in 0.11.3](https://raw.githubusercontent.com/undergroundwires/privacy.sexy/0.11.3/package-lock.json)) . This dependency bump was unintended as side effect of introducing new (ESlint) and bumping other development dependencies (Vue CLI).

According to electron-userland/electron-builder#4793 updating this dependency to pre-release version fixes the issue. Or we could downgrade it to working version to not risk using beta software. I will issue a new patch fixing this along with some other minor improvements. In meantime I think we should flag those warnings as false positives so Microsoft Defender draw better conclusions.

**Comment by undergroundwires** [comment]
Fixed in [0.11.4](https://github.com/undergroundwires/privacy.sexy/releases/tag/0.11.4)
